Blog post on import machines from GitHub

mellson opened the pull request:
This PR adds a blog post on import machines from GitHub.

gavination commented on content/posts/2023-02-06-github-import-machines.mdx:
```diff
@@ -0,0 +1,49 @@
+
+## How to
+
+Importing a machine is easy. You open a file containing one- or more machines on GitHub. Next, you modify the URL; replace `.com` with `.stately.ai`, and the import will start.
```
"one or more" is probs what you meant here. looks like a slight typo.

gavination replied:
use a comma instead of a semicolon after `modify the URL`

gavination commented on content/posts/2023-02-06-github-import-machines.mdx:
```diff
@@ -0,0 +1,49 @@
+
+### Example
+
+If you have a machine at GitHub: `https://github.com/username/repo/blob/main/apps/superMachine.ts`, you update the URL to `https://github.stately.ai/username/repo/blob/main/apps/superMachine.ts` and the import will start 🚀
```
```suggestion
If you have a machine hosted at GitHub: `https://github.com/username/repo/blob/main/apps/superMachine.ts`, you update the URL to `https://github.stately.ai/username/repo/blob/main/apps/superMachine.ts` and the import will start 🚀
```

gavination replied:
just another `nit`, i promise

gavination commented on content/posts/2023-02-06-github-import-machines.mdx:
```diff
@@ -0,0 +1,49 @@
+
+## Get started
+
+This feature requires two things; firstly, you need a subscription to the Stately Studio. Lastly, you must install the GitHub integration app. When you import a machine, we will help you with both.
```
```suggestion
This feature requires the following prerequisites:

- A **professional or enterprise** subscription to the Stately Studio. 
- You must install the GitHub integration app. 

When importing a machine, we help you with both of those tasks. 
```

gavination replied:
Think it might be important to call out that this is a paid feature. I also really like bulleted lists for formatting's sake.

gavination commented on content/posts/2023-02-06-github-import-machines.mdx:
```diff
@@ -0,0 +1,49 @@
+
+This feature requires two things; firstly, you need a subscription to the Stately Studio. Lastly, you must install the GitHub integration app. When you import a machine, we will help you with both.
+
+Your GitHub token gets stored on your own devices; we don’t save it in our database. This is to keep your data as safe as possible, the tradeoff being you have to accept the integration on the initial import on each device you use.
```
```suggestion
Your GitHub token is stored on your own devices by design; we don’t save it in our database. This is to keep your data as safe as possible with the tradeoff being that you must accept the integration on the initial import for each device you use.
```

gavination replied:
purely a `nitpick` here. just added some things i thought might flow a tad better

gavination commented on content/posts/2023-02-06-github-import-machines.mdx:
```diff
@@ -0,0 +1,49 @@
+
+### Bookmarklet bonus
+
+You can create a [bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet) if you want to make it extra easy to import machines from GitHub. You do this by adding a new bookmark and setting the web address to `javascript:(function(){ location.href = 'https://github.stately.ai/' + window.location.pathname;})();`. Now you can click the bookmarklet whenever you want to import a machine. The bookmarkl ...
```
Love this pro tip!

gavination reviewed (commented):
This is very well written, @mellson! It's very clear, concise, and I like the 1st person point of view you speak. To me, it reads as very casual. I just added some suggestions for minor tweaks that I think would help it flow a bit better.

mellson commented:
> This is very well written, @mellson! It's very clear, concise, and I like the 1st person point of view you speak. To me, it reads as very casual. I just added some suggestions for minor tweaks that I think would help it flow a bit better.

Thank you for the feedback 😊 I've implemented all your suggestions. When you think it's good to go hit the green button and lets ship this 🚢

gavination commented:
> 

The button has been hit!
